homeassistant/components/homematic/const.py

```diff
@@ -205,6 +205,7 @@
         "IPWeatherSensorBasic",
         "IPThermostatWall",
         "IPThermostatWall2",
+        "CO2SensorIP",
```

Comment:
Is this the only thing in the PR?

Reply:
Yes. That line was missing, which prevents the sensor entity to be created.